src/components/HistoryTable.js

```diff
@@ -71,15 +115,18 @@ class HistoryRow extends Component {
         <tr className="history-row-details"
             style={{display: open ? "table-row" : "none"}}>
           <td colSpan="6">
-            <pre>{JSON.stringify(entry, null, 2)}</pre>
+            {previous
+              ? <Diff source={entry.target} target={previous.target} />
+              : error
+                ? <p className="alert alert-danger">{error}</p>
+                : <p className="alert alert-info">This resource has no previous versions.</p>}
```

Comment:
When clicking on the details of create, we may want to keep `  <pre>{JSON.stringify(entry, null, 2)}</pre>` no?